src/main/java/com/snowplowanalytics/refererparser/Parser.java

```diff
@@ -23,73 +23,58 @@
 import java.nio.charset.Charset;
 import java.io.IOException;
 import java.io.InputStream;
+import java.util.ArrayList;
 import java.util.List;
 import java.util.Map;
 import java.util.HashMap;
 import java.util.Collections;
-
-// SnakeYAML
-import org.yaml.snakeyaml.Yaml;
-import org.yaml.snakeyaml.constructor.SafeConstructor;
+import java.util.Iterator;
 
 // Apache URLEncodedUtils
 import org.apache.http.NameValuePair;
 import org.apache.http.client.utils.URLEncodedUtils;
 
+import com.snowplowanalytics.refererparser.scala.ParseReferersJson;
+
 /**
  * Java implementation of <a href="https://github.com/snowplow/referer-parser">Referer Parser</a>
  *
  * @author Alex Dean (@alexatkeplar) <support at snowplowanalytics com>
  */
 public class Parser {
 
-  private static final String REFERERS_YAML_PATH = "/referers.yml";
+  private static final String REFERERS_JSON_PATH = "/referers.json";
```

Comment:
I actually think we should completely remove it from the JAR, wdyt @alexanderdean ?